tests/acceptance/tooltip-auto-close-test.js

```diff
@@ -0,0 +1,42 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from '../helpers/start-app';
+
+var application;
+
+module('Acceptance | tooltip auto close', {
+  beforeEach: function() {
+    application = startApp();
+  },
+
+  afterEach: function() {
+    Ember.run(application, 'destroy');
+  }
+});
+
+test('Auto-closing tooltips using duration parameter', function(assert) {
+  visit('/tooltip-auto-close');
+
+  assert.expect(25);
+
+  assertTooltipProperties(assert, 'auto-close-basic', {
```

Comment:
As far as I can tell, ember's async test helpers are aware of the `Ember.run.later` call in the code. So, yes the auto-closing is being tested (or rather, the tooltip's absence is being asserted).

You can alter the duration attrs provided, and the test suite will take longer/shorter according to change.

Really cool!